pkg/internal/validation/dns.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	dns1123LabelFmt     string = "[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
-	dns1123SubdomainFmt string = dns1123LabelFmt + "(\\." + dns1123LabelFmt + ")*"
+	dns1123SubdomainFmt string = dns1123LabelFmt + "(?:\\." + dns1123LabelFmt + ")*"
```

Comment:
It changes the validation. Could you please clarify why this change was made?

Reply:
It is changing the capture block into a non-capture block. You can see how every other group is a non-capture block: 

```
dns1123LabelFmt     string = "[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
```

This means that the regex matcher won't store that value in a variable, because we don't use it. All groups were change to non-capture groups in a recent PR but this one seems to have been missed.